org.eclipse.m2e.pde.target/src/org/eclipse/m2e/pde/target/MavenTargetLocation.java

```diff
@@ -334,6 +332,9 @@ private File unpackFeatureFile(Artifact artifact, CacheManager cacheManager) {
 
 	private void addBundleForArtifact(Artifact artifact, CacheManager cacheManager, IMaven maven,
 			TargetBundles targetBundles, IProgressMonitor monitor) {
+		if (isExcluded(artifact)) {
```

Comment:
Excluded items must be part of the `TargetBundles` as we want to still show them as ignored in the UI!